tbtc-ts/test/data/proof.ts

```diff
@@ -0,0 +1,347 @@
+import {
+  Proof,
+  RawTransaction,
+  Transaction,
+  TransactionMerkleBranch,
+} from "../../src/bitcoin"
+
+/**
+ * Represents a set of data used for given proof scenario.
+ */
+export interface ProofTestData {
+  requiredConfirmations: number
+  clientData: {
```

Comment:
What's the reasoning behind naming it `clientData`? What is the "client" here?

Reply:
It's data that that needs to be returned from the bitcoin client mock, as opposed to the expected data, or data that drives the test (arguments).

Reply:
Same as in https://github.com/keep-network/tbtc-v2/pull/86#discussion_r822568649

Reply:
Done.